domenetjenester/beregningsgrunnlag/src/main/java/no/nav/foreldrepenger/domene/migrering/BeregningMigreringTjeneste.java

```diff
@@ -149,6 +156,11 @@ private void migrerBehandling(Behandling behandling) {
                 oppdaterKoblingMedStpGrunnbeløpOgReguleringsbehov(kobling, response.grunnlag().getBeregningsgrunnlag());
             }
 
+            var førsteUttaksdato = skjæringstidspunktTjeneste.getSkjæringstidspunkter(ref.behandlingId()).getFørsteUttaksdato();
```

Comment:
Her kan du potensielt få noen exceptions for tomme uttak. Da er første uttaksdato passe udefinert ....

Reply:
Her er det vel aktuelt med både løpende migrering + å ettermigrere de med beregningsgrunnlag.stp >= 1/5-2 uker. Enkelt utvalg.

Reply:
@jolarsen Ble dette rett da?